src/main/java/org/openmrs/addonindex/domain/artifactory/VersionList.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+
+package org.openmrs.addonindex.domain.artifactory;
+
+import org.openmrs.addonindex.util.Version;
+
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.SortedSet;
+import java.util.TreeSet;
+
+public class VersionList {
+
+    private SortedSet<Version> versions;
+
+    private static final String[] STRINGS_TO_EXCLUDE = {"alpha", "beta", "hibernate", "maven", "RC", "SNAPSHOT"};
```

Comment:
I would get rid of hibernate, unless we really need it.
To be more clear, you can exclude "maven.xml" (instead of just "maven"). But actually, couldn't we exclude things before they even get to this class if we include only where ArtifactoryFolderInfoChild.folder == true?